tests/upgrades/conftest.py

```diff
@@ -160,6 +161,13 @@ def get_entity_data(scenario_name):
     return entity_data
 
 
+def get_all_entity_data():
+    """Fetches the dictionary of entities"""
```

Comment:
Please extend the docstring for details as to what entities, when saved and when to be retrived.